modules/configuration/src/test/java/org/apache/ignite/internal/configuration/tree/InternalIdTest.java

```diff
@@ -98,8 +98,7 @@ void setUp() {
         );
 
         registry.start();
-
-        registry.initializeDefaults();
+        registry.persistDefaults().get(1, TimeUnit.SECONDS);
```

Comment:
Why don't we move this call inside of the `start()`? It may return the future as well. I see that they are always called together, that's a good argument for combining them.